### ISSUE TITLE
Add support for plain text notes

### ISSUE THREAD

**Issue by zbraniecki** [Fmstrat/ownnote#246]
It seems that there's no way to work with plain text notes, which makes it unnecessary hard to work with files in `./Notes` directory for a user who doesn't want rich formatting.

It would be awesome to be able to select default mode as plain text and just use simple text files.

**Comment by Fmstrat** [comment]
I will put this on the enhancement list, but it's going to be a low priority. Most of the reason people use this over the default "Notes" app is for the rich formatting.

**Comment by zbraniecki** [comment]
Yeah, understandable. But in case of my provider, I don't think I got any choice ;)

**Comment by blvb** [comment]
Please raise priority. Start simple: just list TXT files and honor CRs.
